README.md

```diff
@@ -1,21 +1,57 @@
-# einsum-experiment
-Experimental implementation of torch/onnx einsum
+# einsum
+> Experimental implementation of torch/onnx einsum
+
+## Install
+
+```bash
+pip3 install -U einsum
+```
+
+Or develop locally:
 
-pyproject.toml was filled in following [setuptools quickstart instructions](https://setuptools.pypa.io/en/latest/userguide/quickstart.html),
-which makes it possible to generate a distribution with
 ```bash
-python3 -m pip build
-python3 -m build
+https://github.com/sorenlassen/einsum-experiment ~/einsum
```

Comment:
should there be `git clone` in front of this?

Reply:
Good catch! I accidentally deleted it when I was copying your repo url.

Reply:
Fixed!